schema/dex/view_trades_alpha.sql

```diff
@@ -227,6 +227,56 @@ FROM (
         evt_index
      FROM zeroex_v3."Exchange_evt_Fill" fills
 
+    UNION
+
+    -- dYdX Solo Margin v2
+         SELECT
```

Comment:
The `UNION` and `SELECT` should be idented at the same level as the rest of the query